### ISSUE TITLE
Froms window size is excceding the screen

### ISSUE THREAD

**Issue by GeorgeYaz** [espanso/espanso#1561]
Hi,

I noticed that if you want to replace a trigger with a From consisting of a long sentence, then the size of the Form Window would exceed the size of the screen.

I tried to add the following to the default.yml, but it didn't work out:
    form_size:
      width: 400
      height: 300

Is there any feature to wrap the text within the Form without creating new lines.

Thank you for your time.

Best,
- George

**Comment by contraexemplo** [comment]
The only workaround I could find was to completely detach what's shown on the Espanso form window from what's pasted by Espanso when I'm done filling out a form.

I was reading the form documentation looking for answers when I got to [this section about controls with verbose syntax](https://espanso.org/docs/matches/forms/#controls-with-the-verbose-syntax). It uses `replace` instead of `form` right after the `trigger` parameter, and it controls what's displayed on the Espanso window through a `layout` parameter.

I was able to use `\n` to add line breaks on the text displayed by Espanso, and that decreased the window size significantly. It isn't exactly what you want (automatic text wrap—that's exactly what I wanted too), but it's an okay-ish short to medium term solution while text wrap isn't implemented.
